### PR TITLE
tgui: Exosuit Control Console

### PR DESCRIPTION
**Pull request by Arkatos1**
## About The Pull Request

This PR converts Exosuit Control Console from html based interface to tgui with some tweaks and improvements.

**OLD:**

exosuitOLD

**NEW (Updated):**

ExoUpdated

## Why It's Good For The Game

Better UI.

## Changelog
:cl: Arkatos
add: Exosuit Control Console now uses tgui.
/:cl:

### CODE REVIEW

**Comment by GunHog** [comment]
This is excellent! The only nitpick is that the EMP button should either read "EM Pulse" or "EMP", as "EMP Pulse" = "ElectroMagnetic Pulse Pulse".

File: tgui/packages/tgui/interfaces/ExosuitConsole.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            </LabeledList.Item>
+            <LabeledList.Item label="Airtank">
+              {mech.airtank
+                ? <AnimatedNumber
+                  value={mech.airtank}
+                  format={value => toFixed(value, 2) + ' kPa'} />
+                : "Not Equipped"}
+            </LabeledList.Item>
+            <LabeledList.Item label="Pilot">
+              {mech.pilot ? mech.pilot : "None"}
```
Comment (stylemistake) [inline tgui/packages/tgui/interfaces/ExosuitConsole.js]: Write `mech.pilot || 'None'` to not repeat yourself.

**Comment by Tharcoonvagh** [comment]
*sees exosuit in the title*
Oh neat, the exofab gets a normal ui now!

File: tgui/packages/tgui/interfaces/ExosuitConsole.js (hunk 1 above)
Comment (Arkatos1) [reply tgui/packages/tgui/interfaces/ExosuitConsole.js]: Done.

File: tgui/packages/tgui/interfaces/ExosuitConsole.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      <Section>
+        <NoticeBox textAlign="center">
+          No exosuits detected
+        </NoticeBox>
+      </Section>
+    );
+  }
+
+  return (
+    <Section>
```
[the remark this answers is not in the document]
Comment (Arkatos1) [reply tgui/packages/tgui/interfaces/ExosuitConsole.js]: Nested sections removed.

File: tgui/packages/tgui/interfaces/ExosuitConsole.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                {mech.integrity}%
+              </Box>
+            </LabeledList.Item>
+            <LabeledList.Item label="Charge">
+              <Box color={mech.charge <= 30
+                ? 'bad'
+                : mech.charge <= 70
+                  ? 'average'
+                  : 'good'}>
+                {mech.charge ? mech.charge + "%" : "Not Found"}
```
[the remark this answers is not in the document]
Comment (Arkatos1) [reply tgui/packages/tgui/interfaces/ExosuitConsole.js]: I have implemented the checks suggested, and it works as exactly as you thought - if the cell is missing, value gets casted to zero and "Not Found" was in `bad` color, which is exactly what is wanted there. Thanks!

**Comment by Arkatos1** [comment]
EMP button description clarified, all review suggestions implemented and tested. Main post updated with new image.

**Review by stylemistake — APPROVED** [review]

**Pull request merged**